staging/src/k8s.io/client-go/go.mod

```diff
@@ -13,6 +13,7 @@ require (
 	github.com/google/go-cmp v0.5.9
 	github.com/google/gofuzz v1.2.0
 	github.com/google/uuid v1.3.0
+	github.com/gorilla/websocket v1.4.2
```

Comment:
if we're adding more callers to this, drop it from hack/unwanted-dependencies.json

```
"github.com/gorilla/websocket": "unmaintained, archive mode",
```

Reply:
Done